modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectEntryModelDocumentContributor.java

```diff
@@ -95,6 +110,27 @@ private void _contribute(Document document, ObjectEntry objectEntry)
 		document.addKeyword(
 			"objectDefinitionId", objectEntry.getObjectDefinitionId());
 
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinition(
+				objectEntry.getObjectDefinitionId());
+
+		document.addKeyword(
+			"objectDefinitionName", objectDefinition.getShortName());
+
```

Comment:
this is the section of code related to the "title" fields

Reply:
@marco-leo is working on create a configuration on Object Definition admin UI allowing user to select which Object Field represents the title of it